assets/js/components/Household/create-household.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+    <div class="household">
+        <input v-model="name"/>
+        <button @click.prevent="createHousehold">Create Household</button>
+    </div>
+</template>
+<script>
+import axios from "axios";
+
+export default {
+  name: "household",
+  inject: ["api"],
+  methods: {
+    createHousehold: function() {
+      axios
+        .post(
+          `${this.api}/households`,
+          {
+            household: {
+              name: this.name
+            }
+          },
+          {
+            headers: { Authorization: `Bearer ${this.$store.getters.auth}` }
```

Comment:
Can come in a future PR, but we should probably add a mixin of some sort to automatically add this header to requests